AFQ/registration.py

```diff
@@ -367,7 +367,6 @@ def register_series(series, ref, pipeline):
 
 
 def register_dwi(data_files, bval_files, bvec_files,
-                 b0_ref=0,
```

Comment:
Good catch. Maybe instead of eliminating this, replace the `ref=0` input [below](https://github.com/yeatmanlab/pyAFQ/pull/507/files#diff-08f80bcf98823634b60e56a12f8d9110R393) with `ref=b0_ref`?